api/sds.proto

```diff
@@ -93,6 +93,9 @@ message CertificateValidationContext {
 
   // Must present signed certificate time-stamp.
   google.protobuf.BoolValue require_signed_certificate_timestamp = 6;
+
+  // If specified, Envoy will verify certificates against this CRL.
```

Comment:
Are there multiple possible formats (anything besides PEM)?  If so, please document the expected format.